### ISSUE TITLE
cb is not a function when running ssr:serve

### ISSUE THREAD

**Issue by Codermar** [vueneue/vueneue#31]
I am getting:
```
(node:28153) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
✖ ｢wdm｣: TypeError: cb is not a function
    at tryHandleCache (/usr/local/playground/vuejs/vue-cli-ssr-one/node_modules/ejs/lib/ejs.js:226:12)
    at Object.exports.renderFile (/usr/local/playground/vuejs/vue-cli-ssr-one/node_modules/ejs/lib/ejs.js:437:10)
    at NeueCorePlugin.writeBoot (/usr/local/playground/vuejs/vue-cli-ssr-one/node_modules/@vueneue/ssr-core/build/NeueCorePlugin.js:52:28)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:745:11)
    at startup (internal/bootstrap/node.js:266:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:596:3)
```

1) I started a new project with the cli. (see preset at the bottom
  - (I picked the option use history mode for router)
2) I run vue add @vueneue/ssr but get the error above when I run ssr:serve

```
? Add a production ready Dockerfile? Yes

🚀  Invoking generator for @vueneue/vue-cli-plugin-ssr...
📦  Installing additional dependencies...

added 95 packages from 115 contributors, updated 1 package and audited 22656 packages in 11.376s
found 0 vulnerabilities

✔  Successfully invoked generator for plugin: @vueneue/vue-cli-plugin-ssr
```

Preset used:

```json
{
  "packageManager": "npm",
  "useTaobaoRegistry": false,
  "presets": {
    "Full preset One": {
      "useConfigFiles": true,
      "plugins": {
        "@vue/cli-plugin-babel": {},
        "@vue/cli-plugin-eslint": {
          "config": "airbnb",
          "lintOn": [
            "save"
          ]
        },
        "@vue/cli-plugin-unit-mocha": {},
        "@vue/cli-plugin-e2e-nightwatch": {}
      },
      "router": true,
      "routerHistoryMode": true,
      "vuex": true,
      "cssPreprocessor": "sass"
    },
    "Full preset": {
      "useConfigFiles": true,
      "plugins": {
        "@vue/cli-plugin-babel": {},
        "@vue/cli-plugin-eslint": {
          "config": "airbnb",
          "lintOn": [
            "save"
          ]
        },
        "@vue/cli-plugin-unit-mocha": {},
        "@vue/cli-plugin-e2e-nightwatch": {}
      },
      "router": true,
      "routerHistoryMode": true,
      "vuex": true,
      "cssPreprocessor": "sass"
    }
  }
}
````

**Comment by chymz** [comment]
It seems that npm install an old version of `ejs` package.

As a workaround you can install latest version: `npm install -D ejs@latest`

I'll publish a fix soon

Thanks for testing/reporting !

**Comment by Codermar** [comment]
Thanks.
Got some lint errors after the install. I run the lint task with --fix and got most of them out except for these which I manually corrected, then it run successfully:
error: Missing file extension "vue" for "./AppView" (import/extensions) at src/App.vue:8:21:
   6 |
   7 | <script>
>  8 | import AppView from './AppView';
     |                     ^
   9 |
  10 | export default {
  11 |   components: {


error: Expected to return a value at the end of method 'error' (consistent-return) at src/components/ErrorPage.vue:53:5:
  51 |     },
  52 |
> 53 |     error() {
     |     ^
  54 |       if (this.current && this.current.error) return this.current.error;
  55 |     },
  56 |


error: Expected '!==' and instead saw '!=' (eqeqeq) at src/components/ErrorPage.vue:65:42:
  63 |     gotoHome() {
  64 |       this.$store.commit('errorHandler/CLEAR');
> 65 |       if (this.$router.currentRoute.path != '/') {
     |                                          ^
  66 |         this.$router.replace('/');
  67 |       }
  68 |     },

**Comment by chymz** [comment]
Fixed in `0.3.19`
